pcs/lib/tools.py

```diff
@@ -97,7 +101,7 @@ def write_tmpfile(data, binary=False):
 def get_tmp_file(
     data: Union[None, bytes, str] = None,
     binary: bool = False,
-) -> Any:
+) -> Iterator[Union[IO[str], IO[bytes]]]:
```

Comment:
Based on https://github.com/python/typeshed/issues/2772 we should use `Generator` here

```suggestion
) -> Generator[Union[IO[str], IO[bytes]], None, None]:
```